.travis.yml

```diff
@@ -14,5 +14,5 @@ before_install:
 
 script:
  - ./autogen.sh
- - ./configure --with-stage1-default-name="example.com/some-stage1-for-rkt" --with-stage1-default-version="0.0.1"
+ - ./configure
```

Comment:
SGTM, but please add a comment for posterity

On Fri, Jan 29, 2016 at 5:15 PM, Krzesimir Nowak <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/coreos/rkt/pull/2050#discussion_r51279394>:
>
> > @@ -14,5 +14,5 @@ before_install:
> >
> >  script:
> >   - ./autogen.sh
> > - - ./configure --with-stage1-default-name="example.com/some-stage1-for-rkt" --with-stage1-default-version="0.0.1"
> > + - ./configure
>
> Not sure if I like the change - it will be building three flavors now
> (coreos, kvm and fly). Maybe instead consider building ./configure
> --with-stage1-flavors=host,fly - this should be all the go stuff we have,
> but will not download CoreOS PXE image nor download kernel and kvm and
> build them.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/2050/files#r51279394>.
>